### PR TITLE
Bug 1775765: Addition of skeleton components for dashboard views

### PR DESCRIPTION
**Pull request by sg00dwin**
https://jira.coreos.com/browse/CONSOLE-1908

Dashboard-all-skeleton

### CODE REVIEW

File: frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss [hunk 1]
```diff
@@ -108,13 +107,13 @@
...
 }
 
-.skeleton-alerts {
+.skeleton-health {
   animation: $skeleton-animation;
   background: $skeleton-color;
-  height: 40px;
-  width: 400px;
+  height: $skeleton-bone-height-1line;
+  width: $skeleton-bone-height-1line;
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss]: Nit:  alpha

File: frontend/packages/console-shared/src/styles/skeleton-screen.scss [hunk 2]
```diff
@@ -1,2 +1,53 @@
 $skeleton-animation: loading-skeleton 1s ease-out .15s infinite alternate;
 $skeleton-color: #eaedf1;
+$skeleton-bone-height-1line: 24px;
+
+// Dashboard Status
+$skeleton-status-bone-height: $skeleton-bone-height-1line;
+$skeleton-status-bone: linear-gradient($skeleton-color $skeleton-status-bone-height, transparent 0);
+$skeleton-status-position: 0 10px;
```
Comment (rhamilto) [inline frontend/packages/console-shared/src/styles/skeleton-screen.scss]: NIt throughout file:  alpha

File: frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss [hunk 3]
```diff
@@ -124,17 +123,37 @@
...
+  height: 50px;
+  margin-bottom: 1em;
   opacity: 0;
+  width: 100%;
   &::after {
     background-repeat: no-repeat;
-    content: "";
-    display: block;
+    content:"";
+    display:block;
```
Comment (TheRealJon) [inline frontend/packages/console-shared/src/components/dashboard/status-card/status-card.scss]: nit: space after ":"

File: frontend/packages/console-shared/src/styles/skeleton-screen.scss [hunk 4]
```diff
@@ -1,2 +1,53 @@
...
+  &::after {
+    background-repeat: no-repeat;
+    content: "";
+    display: block;
+    height: 100%;
+    width: 100%;
+  }
+}
+
+.dashboard-chart {
```
Comment (spadgett) [inline frontend/packages/console-shared/src/styles/skeleton-screen.scss]: `skeleton-chart` maybe? The charts don't always appear in the dashboard, and it's not clear that the class is for a skeleton.

**Comment by spadgett** [comment]
@sg00dwin feel free to remove WIP if this is ready

**Pull request merged**